app/views/host_mailer/error_state.html.erb

```diff
@@ -1,4 +1,4 @@
-<h2 class="headline"><%= _("<b>Foreman</b> Puppet error report").html_safe %></h2>
+<h2 class="headline"><%= _("<b>Foreman</b> configuration management error report").html_safe %></h2>
```

Comment:
Mail subject has capitals while this subject not. Just an observation, same below.